jax_distribution/requirements.txt

```diff
@@ -1,14 +1,12 @@
-jax>=0.2.26
-jaxlib>=0.1.74
-gymnax
-evosax
+jax<=0.4.13
+jaxlib<=0.4.13
```

Comment:
Why less than, is there something wrong with 0.4.14?

Reply:
Indeed, 0.4.14 gives the following error: 
`AttributeError: module 'jax' has no attribute 'xla'`